Windows 32b failure: Error copying file drconfiglib.dll

Seeing this failure in #5717 

```
2023-02-27T04:29:11.5101083Z ==================================================
2023-02-27T04:29:11.5101644Z RESULTS
2023-02-27T04:29:11.5101998Z 
2023-02-27T04:29:11.5102315Z debug-internal-32: **** 1 build errors ****
2023-02-27T04:29:11.5103319Z 	Error copying file "/.../dynamorio/build_debug-internal-32/lib32/debug/../drconfiglib.dll" to "/.../dynamorio/build_debug-internal-32/suite/tests/bin/drconfiglib.dll".
2023-02-27T04:29:11.5103853Z 
2023-02-27T04:29:11.5470204Z Finished running ctest --output-on-failure -VV -S "D:/a/dynamorio/dynamorio/suite/runsuite.cmake,automated_ci;use_ninja;32_only;debug_only"
2023-02-27T04:29:11.5470987Z RESULTS
2023-02-27T04:29:11.5471336Z 
2023-02-27T04:29:11.5471487Z 
2023-02-27T04:29:11.5471769Z ====> FAILURE in debug-internal-32 <====
2023-02-27T04:29:11.5472812Z debug-internal-32: **** 1 build errors ****
2023-02-27T04:29:11.5473596Z 	Error copying file "/.../dynamorio/build_debug-internal-32/lib32/debug/../drconfiglib.dll" to "/.../dynamorio/build_debug-internal-32/suite/tests/bin/drconfiglib.dll".
2023-02-27T04:29:11.5562265Z ##[error]Process completed with exit code 1.
```

Data point: This fails on 64 bit too. Possibly the same race.
https://github.com/DynamoRIO/dynamorio/actions/runs/10894111956/job/30230273364#step:6:1407
```
RESULTS

debug-internal-64: **** 1 build errors ****
	Error copying file "/.../dynamorio/build_debug-internal-64/lib64/debug/../drconfiglib.dll" to "/.../dynamorio/build_debug-internal-64/suite/tests/bin/drconfiglib.dll".

Finished running ctest --output-on-failure -VV -S "D:/a/dynamorio/dynamorio/suite/runsuite.cmake,automated_ci;use_ninja;64_only;debug_only"
RESULTS


====> FAILURE in debug-internal-64 <====
debug-internal-64: **** 1 build errors ****
	Error copying file "/.../dynamorio/build_debug-internal-64/lib64/debug/../drconfiglib.dll" to "/.../dynamorio/build_debug-internal-64/suite/tests/bin/drconfiglib.dll".
Error: Process completed with exit code 1.
```

--- a/suite/tests/CMakeLists.txt
+++ b/suite/tests/CMakeLists.txt
@@ -970,15 +970,19 @@ function(add_api_exe test source use_decoder use_static_DR)
         COMMAND ${CMAKE_COMMAND} ARGS
           -E touch "${drcopy_stamp}"
         COMMAND ${CMAKE_COMMAND} ARGS
-          -E copy "${MAIN_LIBRARY_OUTPUT_DIRECTORY}/dynamorio.dll"
+          -E copy_if_different "${MAIN_LIBRARY_OUTPUT_DIRECTORY}/dynamorio.dll"
           "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dynamorio.dll"
         # libutil.drconfig_test needs drconfiglib.
         COMMAND ${CMAKE_COMMAND} ARGS
-          -E copy "${MAIN_LIBRARY_OUTPUT_DIRECTORY}/../drconfiglib.dll"
+          # XXX i#5888: There is a race somewhere causing this copy to sometimes
+          # fail, presumably because another target already copied it.  We use
+          # copy_if_different to avoid failure if it exists, here and in the other
+          # copy commands above and below.
+          -E copy_if_different "${MAIN_LIBRARY_OUTPUT_DIRECTORY}/../drconfiglib.dll"
           "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/drconfiglib.dll"
         # We copy drsyms.dll too, for api.symtest
         COMMAND ${CMAKE_COMMAND} ARGS
-          -E copy "${PROJECT_BINARY_DIR}/ext/${INSTALL_LIB}/drsyms.dll"
+          -E copy_if_different "${PROJECT_BINARY_DIR}/ext/${INSTALL_LIB}/drsyms.dll"
           "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/drsyms.dll"
         VERBATIM)
       add_custom_target(drcopy DEPENDS "${drcopy_stamp}")

